src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -32,7 +32,7 @@ azure-mgmt-cognitiveservices==13.5.0
 azure-mgmt-compute==30.4.0
 azure-mgmt-containerinstance==10.1.0
 azure-mgmt-containerregistry==10.1.0
-azure-mgmt-containerservice==28.0.0
+azure-mgmt-containerservice==29.0.0
```

Comment:
https://pypi.org/project/azure-mgmt-containerservice/29.0.0

Python SDK with 2023-11-01 API, see https://github.com/Azure/sdk-release-request/issues/4847

Reply:
@FumingZhang ok to just bump the SDK version here as part of this PR?

Reply:
Apparently not, getting this [error](https://dev.azure.com/azclitools/public/_build/results?buildId=124273&view=logs&j=537f1e04-c6d5-53b2-3a1f-f7387cc6e02d&t=cae9a690-b596-5b1d-0d0e-9efe890cb6db&l=441):

```
The conflict is caused by:
    The user requested azure-mgmt-containerservice==29.0.0
    azure-cli 2.56.0 depends on azure-mgmt-containerservice~=28.0.0
```

Reply:
I bumped the SDK in PR #28229. Please rebase from main branch to pick up the change.

Reply:
Done, thank you!